crates/static-analysis-kernel/src/classifiers/tests.rs

```diff
@@ -303,6 +369,26 @@ mod cfg_test_tests {
         assert!(!trie_has_prefix(&trie, "no.sequence.match".split(".")));
     }
 
+    #[test]
+    fn language_csharp() {
+        let shoulds = &[
+            "using Xunit;",
+            "using UT = Microsoft.VisualStudio.TestTools.UnitTesting;",
+        ];
+        let should_nots = &["using System.Diagnostics;"];
```

Comment:
nit(non-blocking): a good test case might be to ensure we never consider the alias, e.g:

```cs
using Xunit = CustomFramework;
```

though it's not necessary at all